### ISSUE TITLE
Feature Request: Clippy

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#935]
My users have bad Galaxy habits. This is a combination of
- lack of training
- generally not putting all of the effort needed into maintaining tidy workspaces

That's normal, and I can combat the lack of training, but I'm having more trouble fixing their digital behaviours. I came to the conclusion I needed to nag them regularly regarding best practices, because we all need a gentle reminder now and then (myself included) of where we can change our habits to make life easier later. Of _course_ I remembered clippy! 

![clippy1](https://cloud.githubusercontent.com/assets/458683/10548260/f83b886c-73fe-11e5-8eb4-110f9f631e1e.png)

So, I'm suggesting implementing a Galaxy clippy (maybe with a _Galactic explorer_ (*groan*, I know) like an astronomer or spaceman in a little spacesuit) to help users find their way around Galaxy. They should suggest things like:
- Naming your histories
- Using workflows
- Telling users about search boxes if they scroll around histories a lot

This is sort of tangential to requests for introductory tours in that it's a more regular/gentler "hey, listen, you might want to consider X"

**Comment by bgruening** [comment]
This can be easily done in a potential Galaxy introduction tour :)

**Comment by hexylena** [comment]
@bgruening yes, but I want _constant_ nagging, not just for new users :P Old users who've dismissed the tours/been using galaxy for a long time should all get nagged about choosing better names for histories.

Cute avatar giving the introduction would be great though.

**Comment by nturaga** [comment]
@erasche  We have been working on a tool similar to this. We even have a pretty awesome "artwork" thanks to James' postdoc Mallory. I'll inform you more about this during our hangout session on Monday.

**Comment by hexylena** [comment]
@nitesh1989 awesome, I'll look forward to it :)

**Comment by erasche** [comment]
Another thing I'd love for clippy to do is to remind users how to submit bug reports when tools fail. 

My users have this knee-jerk reaction to email me and say "Hey Eric, my tool failed." which then wastes time as I have to re-explain that I need them to click the button, have to impersonate them + browse through their histories to find the right one and then submit the bug report for them. If they could be told "hey, submit a bug repotr here" and "an admin will be in touch with you shortly" to hopefully short circuit the unnecessary emails from them.

**Comment by hexylena** [comment]
https://www.smore.com/clippy-js

**Comment by hexylena** [comment]
@jennaj supports this, she says in a telco and proposes DNA. I seconded with Jurassic parks' "Mr DNA" as our clippy

![image](https://user-images.githubusercontent.com/458683/150378474-f45a4547-ca89-45a5-ad12-0aa9f097b356.png)
